README.md

```diff
@@ -45,3 +45,32 @@ This project uses [sphinx](https://www.sphinx-doc.org/) to generate the document
 ~ make html
 ```
 The built documentation will be in `docs/_build/html`
+
+### Editor configuration
```

Comment:
Shouldn't this be at the same level of `Generating documentation`? It being a level under generating documentation doesn't seem correct.

Reply:
Good catch. I misread the number of `#`s on "Generating documentation" and thought it was under "Development".